Command item
---
int32 item_index